plasmapy/physics/quantum.py

```diff
@@ -281,3 +284,157 @@ def Thomas_Fermi_length(n_e):
     energy_F = Fermi_energy(n_e)
     lambda_TF = np.sqrt(2 * eps0 * energy_F / (3 * n_e * e ** 2))
     return lambda_TF.to(units.m)
+
+
+@check_quantity({
+    'n_e': {'units': units.m**-3, 'can_be_negative': False}
+})
+def Wigner_Seitz_radius(n):
+    r"""Calculate the Wigner-Seitz radius, which approximates the inter-
+    particle spacing. It is the radius of a sphere whose volume is 
```

Comment:
"inter-particle spacing" under what conditions and assumptions? Sorry, it's my first time with this function.

Reply:
It's sort of described in the docstring. You take the density and figure out the average radius between particles. Unit wise you are going from cm^-3 to cm, which is easy enough since for a sphere you have `4/3 * pi * r ** 3 = 1/n` and then just rearrange to solve for `r` the inter-particle spacing. In other words, the volume "occupied" by a particle is the inverse of the density.

Thoughts on how to make this clearer?